packages/manager/modules/pci/src/projects/project/project.html

```diff
@@ -1,37 +1,49 @@
-<div data-ui-view>
-    <div data-ng-if="$ctrl.project">
-        <oui-page-header heading="{{ ::$ctrl.project.description }}">
-            <oui-header-tabs class="mt-3">
-                <oui-header-tabs-item
-                    text="{{ ::'pci_projects_project_home' | translate }}"
-                    ui-sref="pci.projects.project({ serviceName: $ctrl.project.project_id })"
-                    active="$ctrl.$state.is('pci.projects.project')">
-                </oui-header-tabs-item>
-                <oui-header-tabs-item
-                    text="{{ ::'pci_projects_project_parameters' | translate }}"
-                    ui-sref="pci.projects.project.edit({ serviceName: $ctrl.project.project_id })"
-                    active="$ctrl.$state.is('pci.projects.project.edit', { serviceName: $ctrl.project.project_id })">
-                </oui-header-tabs-item>
-            </oui-header-tabs>
-        </oui-page-header>
-        <div class="py-5" data-ui-view="contentTab">
-            <div class="container-fluid px-0">
-                <div class="row">
-                    <div class="col-sm-6 col-lg-4 col-xl mb-4" data-ng-repeat="action in $ctrl.actions track by $index">
-                        <a class="text-center w-100 d-block oui-tile py-4"
-                            data-ui-state="action.state"
-                            data-ui-state-params="{projectId: $ctrl.project.project_id}">
-                            <span class="oui-icon oui-icon-add" aria-hidden="true"></span>
-                            <div class="mt-3 w-100 text-truncate" data-ng-bind=":: action.translation | translate"></div>
-                        </a>
+<div class="container-fluid h-100">
+    <div class="row h-100">
+        <div class="col-md-2 p-0 h-100 d-none d-md-flex flex-column">
+            <cloud-sidebar></cloud-sidebar>
```

Comment:
Pay attention: `<cloud-sidebar>` will be no more available (as standalone perspective).

ref:

```sh
$ git grep "'cloudSidebar'"
packages/manager/apps/public-cloud/src/sidebar/index.js:  .component('cloudSidebar', {
```

Reply:
Oops! Missed that from the GitHub UI.

 `packages/manager/apps/public-cloud/src/sidebar/project-list/project-list.component.js → packages/manager/modules/pci/src/projects/project/sidebar/project-list/project-list.component.js`

Do not pay attention to my previous comment.